src/main/scala/org/broadinstitute/dsde/rawls/user/UserService.scala

```diff
@@ -62,34 +63,35 @@ object UserService {
 
 class UserService(protected val userInfo: UserInfo, dataSource: DataSource, protected val gcsDAO: GoogleServicesDAO, containerDAO: GraphContainerDAO, userDirectoryDAO: UserDirectoryDAO)(implicit protected val executionContext: ExecutionContext) extends Actor with AdminSupport with FutureSupport {
   override def receive = {
-    case SetRefreshToken(token) => setRefreshToken(token) pipeTo context.parent
-    case GetRefreshTokenDate => getRefreshTokenDate() pipeTo context.parent
-
-    case CreateUser => createUser() pipeTo context.parent
-    case AdminGetUserStatus(userRef) => adminGetUserStatus(userRef) pipeTo context.parent
-    case UserGetUserStatus => userGetUserStatus() pipeTo context.parent
-    case EnableUser(userRef) => enableUser(userRef) pipeTo context.parent
-    case DisableUser(userRef) => disableUser(userRef) pipeTo context.parent
-    case ListUsers => listUsers pipeTo context.parent
-    case ImportUsers(rawlsUserInfoList) => importUsers(rawlsUserInfoList) pipeTo context.parent
-    case GetUserGroup(groupRef) => getUserGroup(groupRef) pipeTo context.parent
+    case SetRefreshToken(token) => setRefreshToken(token) pipeTo sender
```

Comment:
To allow service to service communication all pipes here and in WorkspaceService go to sender instead of parent. This way is a little more flexible.